src/utils/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum AssetHash {
+  BaseAsset = '5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225',
```

Comment:
You should have a mapping that maps the coin like L-BTC to an asset hash.

So basically, if you know a backedn supports L-BTC - USDT, you know which `Market` you need to pick.

Reply:
Fixed this.